openssl/src/ssl/mod.rs

```diff
@@ -282,6 +321,12 @@ impl SslContext {
         }
     }
 
+    pub fn set_read_ahead(&self) {
+        unsafe {
+            ffi::SSL_CTX_ctrl(*self.ctx, ffi::SSL_CTRL_SET_READ_AHEAD, 1, ptr::null_mut());
```

Comment:
@sfackler it's also states that the reason for not calling directly is that
there are C macros which wrap it.

@manuels As a better approach I'd recommend to create a function with the
same name as OpenSSL macro in openssl-sys/src/lib.rs - I believe there is a
section already for such wrappers just before huge extern block.